week-1-practise/level-3/1-piping.js

```diff
@@ -11,9 +11,7 @@
 */
 
 var startingValue = 5;
-var result = function() {
-  return "£" + (startingValue * 2 + 10).toFixed(2);
-};
+var result = "£" + (startingValue * 2 + 10).toFixed(2);
```

Comment:
It get's the right answer, but not in quite the way the question is looking for. You have copied what the functions do, but the question would like your to _call_ the functions.

For example, if you only wanted result to be doubled you could have:
```js
var startingValue = 5;
var result = multiply(startingValue, 2);
```

The question would like you to double, add 10 and format in a way like this (and all on the same line).